promotion/promotion.go

```diff
@@ -20,13 +20,21 @@ var (
 		},
 		[]string{"filter", "migrate"},
 	)
+	promotionV2GetCount = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "promotion_v2_get_count",
+			Help: "a count of the number of times the swap promotions were collected",
+		},
+		[]string{"platform"},
+	)
 	promotionExposureCount = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Name: "promotion_exposure_count",
 			Help: "a count of the number of times a single promotion was exposed to clients",
 		},
 		[]string{"id"},
 	)
+	swapSentinelWalletID = "00000000-0000-0000-0000-000000000002"
```

Comment:
Should this be a const, or an environment variable?

Reply:
I made it a const in https://github.com/brave-intl/bat-go/pull/1213/commits/36af29ab409da8c030a17b2330181d5423979a6d